main.py

```diff
@@ -81,5 +81,7 @@ def main(stdscr):
         if ord(key) == 27:
             break
 
+            print("hello")
```

Comment:
What is this?

Reply:
I think I want to commit it somehow and it works